tests/tests.cpp

```diff
@@ -315,7 +315,7 @@ class EngineBasicTest : public testing::Test {
     ASSERT_EQ(DestroyFunc(collection), Status::Ok);
     ASSERT_EQ(PutFunc(collection, key, val), Status::NotFound);
     ASSERT_EQ(GetFunc(collection, key, &got_val), Status::NotFound);
-    ASSERT_EQ(DeleteFunc(collection, key), Status::Ok);
+    ASSERT_EQ(DeleteFunc(collection, key), Status::NotFound);
```

Comment:
Java unit test should also be updated here: https://github.com/pmem/kvdk/blob/main/java/src/test/java/io/pmem/kvdk/EngineTest.java#L94
An example to check status: https://github.com/pmem/kvdk/blob/main/java/src/test/java/io/pmem/kvdk/EngineTest.java#L97

Steps to run Java unit tests (JDK & Maven need to be installed):
```
cd kvdk/build/
cmake .. -DCMAKE_BUILD_TYPE=Release -DCHECK_CPP_STYLE=ON -DWITH_JNI=ON && make -j
cd ../java
mvn clean test
```

Format Java code (on linux since different line endings):
```
mvn spotless:apply
```